test/framework/options.py

```diff
@@ -1441,69 +1441,70 @@ def test_github_copy_ec_from_pr(self):
         # Make sure the test target directory doesn't exist
         remove_dir(test_target_dir)
 
-        all_files_pr8007 = [
-            'Arrow-0.7.1-intel-2017b-Python-3.6.3.eb',
-            'bat-0.3.3-fix-pyspark.patch',
-            'bat-0.3.3-intel-2017b-Python-3.6.3.eb',
+        all_files_pr22365 = [
+            'Environ-3.1-foss-2024a.eb',
+            'QuantumESPRESSO-7.4-foss-2024a-Environ-3.1.eb',
+            'QuantumESPRESSO-7.4-foss-2024a.eb',
+            'QuantumESPRESSO-7.4-parallel-symmetrization.patch',
         ]
 
         # test use of --copy-ec with --from-pr to the current working directory
         cwd = change_dir(test_working_dir)
-        args = ['--copy-ec', '--from-pr', '8007']
+        args = ['--copy-ec', '--from-pr', '22365']
```

Comment:
not ideal, contents of this PR will change once it gets synced with target branch after https://github.com/easybuilders/easybuild-easyconfigs/pull/22345 gets merged...